<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-6VTJ9AE" name="Default test" tests="3" failures="1" timestamp="1 Jul 2019 18:42:12 GMT" time="0.039" errors="0">
  <testcase name="test1" time="0.015" classname="demo.TestNGRetryFailedDemo"/>
  <testcase name="test2" time="0.001" classname="demo.TestNGRetryFailedDemo"/>
  <testcase name="test3" time="0.005" classname="demo.TestNGRetryFailedDemo">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:55)
	at junit.framework.Assert.assertTrue(Assert.java:22)
	at junit.framework.Assert.assertTrue(Assert.java:31)
	at demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:23)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- Default test -->
